packages/x-charts/src/ChartsAxis/ChartsAxis.tsx

```diff
@@ -11,32 +11,37 @@ import {
   ChartsYAxisProps,
 } from '../models/axis';
 import { useXAxes, useYAxes } from '../hooks';
+import { DEFAULT_AXIS_SIZE } from '../constants';
 
 export interface ChartsAxisProps {
   /**
    * Indicate which axis to display the top of the charts.
    * Can be a string (the id of the axis) or an object `ChartsXAxisProps`.
    * @default null
+   * @deprecated Use `xAxis[].position="top"` instead.
```

Comment:
We're suggesting using `xAxis`, but `xAxis` isn't a prop of this component. I'm a bit confused by this, so I suspect a user would be too. Where should I set the `xAxis` prop to maintain the behavior?

Reply:
Yeah, I'm not sure what's the best way to proceed here. I suppose a URL to the migration guide is helpful, at least, so unless we can think of any better option, that seems good to me 👍

Reply:
I'm not super concerned by this point. I guess most of the users are using directly the `ChartsXAxis`/`ChartsYAxis` and they all now where the `xAxis` props goes because with `series` they are the most used